metadata.yml

```diff
@@ -6,13 +6,50 @@ rules:
 mutating: false
 contextAware: false
 annotations:
-  io.kubewarden.policy.title: policy-name
-  io.kubewarden.policy.description: Short description
-  io.kubewarden.policy.author: policy-author
-  io.kubewarden.policy.url: https://github.com/yourorg/policy-name
-  io.kubewarden.policy.source: https://github.com/yourorg/policy-name
+  io.kubewarden.policy.title: psp-hostpaths
+  io.kubewarden.policy.description: PSP that controls usage of hostPath volumes
+  io.kubewarden.policy.author: Kubewarden devs
+  io.kubewarden.policy.url: https://github.com/kubewarden/psp-hostpaths-policy
+  io.kubewarden.policy.source: https://github.com/kubewarden/psp-hostpaths-policy
   io.kubewarden.policy.license: Apache-2.0
   io.kubewarden.policy.usage: |
-    Long explaination.
+    Replacement for the Kubernetes Pod Security Policy that controls the usage of
+    hostPath volumes.
```

Comment:
Can you add an additional line that says something like:

> The policy inspects both the containers and the init containers that are using `hostPath` volumes.

Reply:
True! done in https://github.com/kubewarden/psp-hostpaths-policy/pull/1/commits/ec7b3cc05b34447cf5e275baa28e646589299dc1.